src/admin/components/AddonCard/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { PropTypes } from 'react';
```

Comment:
So I gave Admin its own `AddonCard`. It's nearly line-for-line the same, but otherwise we end up with a weird thing of passing the component it renders in add-on inside. (see: https://github.com/mozilla/addons-frontend/pull/1423#pullrequestreview-10107817)

I'm cool with this approach, as obviously the sharing got weird. Sorry it adds to the lines in this PR, but I think it's the right way going forward.